tla-import/src/main/scala/at/forsyte/apalache/tla/imp/src/SaveToStoreTracker.scala

```diff
@@ -25,4 +25,34 @@ class SaveToStoreTracker(sourceStore: SourceStore) extends TransformationTracker
       }
       output
   }
+
+  /**
+    * Decorate a declaration transformation with a tracker.
+    *
+    * @param tr a declaration transformation
+    * @return a new declaration transformation that applies tr and tracks the changes
+    */
+  override def trackDecl(tr: TlaDeclTransformation): TlaDeclTransformation = {
+    input: TlaDecl =>
+      val output = tr(input)
+      if (output.ID != input.ID) {
+        sourceStore.findOrLog(input.ID).foreach { loc => sourceStore.add(output.ID, loc) }
+      }
+      output
+  }
+
+  /**
+    * A specialized version of trackDecl, which is tuned to TlaOperDecl.
+    *
+    * @param tr a declaration transformation
+    * @return a new declaration transformation that applies tr and tracks the changes
+    */
+  override def trackOperDecl(tr: TlaOperDecl => TlaOperDecl): TlaOperDecl => TlaOperDecl = {
```

Comment:
Why is there a need for a specialized method like this? `trackDecl` should work in all cases where `trackOperDecl` is used.

Reply:
just for convenience, as `TlaOperDecl` is by far the most often used transformation.